gTarrGames/gTarrGames.Web/Views/Person/Update.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@model gTarrGames.Web.Models.UpdatePersonModel
+
+<script src="~/Scripts/DMACC.person.js" type="text/javascript"></script>
+
+<h2>Create Profile</h2>
+
+<div id="successMessage" class="hidden alert alert-success">Successfully Updated!</div>
+<div id="errorMessage" class="hidden alert alert-danger">Error Occurred!</div>
+
+<input id="personId" type="hidden"/>
+<div class="col-md-12">
+    <div class="col-md-6">
+        <label>First Name:</label>
+    </div>
+    <div class="col-md-6">
+        <input id="firstName" type="text" class="form-control"/>
+    </div>
+</div>
+<div class="col-md-12">
+    <div class="col-md-6">
+        <label>Last Name:</label>
+    </div>
+    <div class="col-md-6">
+        <input id="lastName" type="text" class="form-control" />
+    </div>
+</div>
+<div class="col-md-12">
+    <div class="col-md-6">
+        <label>Gender:</label>
+    </div>
+    <div class="col-md-6">
+        <input id="gender" type="text" class="form-control" />
+    </div>
+</div>
+<div class="col-md-12">
+    <div class="col-md-6">
+        <label>Email:</label>
+    </div>
+    <div class="col-md-6">
+        <input id="email" type="text" class="form-control" value=""/>
```

Comment:
To get value from Person.Email here, consider adding:

@model IEnumerable<gTarrGames.Web.Models.UpdatePersonModel>

at top, and using

<input value="@HtmlDisplayNameFor(model => model.Email)">